perllib/FixMyStreet/DB/Result/Problem.pm

```diff
@@ -773,20 +773,23 @@ sub meta_line {
     my $category = $problem->category_display;
     $category = $cobrand->call_hook(change_category_text => $category) || $category;
 
+    # Create display version with quotes for "Other" category
+    my $category_display = ($category eq 'Other') ? '"' . $category . '"' : $category;
+
     # Call a hook on the cobrand to check if the problem should be treated as anonymous
     my $anonymous = $cobrand->call_hook('is_problem_anonymous');
 
     if ( $problem->anonymous || $anonymous ) {
-        if ( $problem->service_display && $category && $category ne _('Other') ) {
+        if ( $problem->service_display && $category && ($category ne _('Other') || $cobrand->show_other_category_in_summary) ) {
```

Comment:
Perhaps factor this Other or flag check out first, as you then use it four times after all